### ISSUE TITLE
Just a question (.net core 3 support)

### ISSUE THREAD

**Issue by quicoli** [XamlFlair/XamlFlair#35]
Hi,

Do you need help to support .net core 3.1 ?

**Comment by XamlFlair** [comment]
Hi @quicoli,

Yes that would be great! I'm busy with two tasks in parallel at the moment:

1) Removing the dependency on Liblog which is now deprecated and replacing logging with `Microsoft.Extensions.Logging.Abstractions`
2) Adding support for the Uno platform

**Comment by XamlFlair** [comment]
@tibel has already done work for adding 3.0 support:

https://github.com/XamlFlair/XamlFlair/pull/27

So hopefully bumping to 3.1 shouldn't be too much work.

**Comment by quicoli** [comment]
Great @XamlFlair I'll do it so.

**Comment by XamlFlair** [comment]
Closing this since I will be doing it soon myself.
